refresh/index.js

```diff
@@ -667,17 +667,16 @@ module.exports = Generator.extend({
     if (!this.fromSwagger) return
 
     return helpers.loadAsync(this.fromSwagger, this.fs)
-      .then(loaded => {
-        return swaggerize.parse(loaded)
-          .then(response => {
-            this.loadedApi = response.loaded
-            this.parsedSwagger = response.parsed
-          })
+      .then(loaded => swaggerize.parse(loaded)
+                        .then(response => {
+                          this.loadedApi = response.loaded
+                          this.parsedSwagger = response.parsed
+                        })
           .catch(err => {
```

Comment:
This is misleadingly indented so that the catch appears to apply to the outer `then`, but it actually belongs to the nested `then`. I think the nested `then` would be better chained rather than nested, in which case the `catch` *would* be attached to the outer `then` (since the nested one would not be nested any more :)).